system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs

```diff
@@ -1002,6 +1049,7 @@ construct_runtime!(
 		Utility: pallet_utility = 40,
 		Multisig: pallet_multisig = 41,
 		Proxy: pallet_proxy = 42,
+		RemoteProxyRelayChain: pallet_remote_proxy = 43,
```

Comment:
Shouldn't we also set `OnSystemEvent` here: https://github.com/polkadot-fellows/runtimes/blob/main/system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs#L684-L686?
```
impl cumulus_pallet_parachain_system::Config for Runtime {
	type RuntimeEvent = RuntimeEvent;
-       type OnSystemEvent = ();
+	type OnSystemEvent = (RemoteProxyRelayChain, );
        ...
```

I also think this feature is a great candidate for adding some end-to-end (e2e) or integration tests. The Chopsticks ecosystem tests PR hasn't been merged yet, but at the very least, this should be achievable with emulated integration tests.

Reply:
Ha! Good find! :D :see_no_evil:

Reply:
Embarrassing xD